streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -265,6 +265,7 @@ public void shouldProcessingFromSinkTopic() {
                  processorSupplier.theCapturedProcessor().processed);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
My opinion is that it's generally better not to suppress but instead just deprecate this method as well. It's not really that important for tests, since no one else is going to call the method, so feel free to take or leave the advice.